documentation/Architecture/Overview.md

```diff
@@ -17,20 +17,20 @@ level, it exposes the following interfaces:
 
 The interfaces depicted in this diagram are
 
-1) The SDK takes as input *data sources*, such as file paths, from the *SDK driver*. The SDK may receive this input when, for example, the user asks 
+1) The SDK receives *data sources* (e.g. file paths) from the *SDK driver*. For example, the SDK may receive this input when the user asks 
```

Comment:
When i read this, it interrupted the flow of the explanation. I thought it might be better to have the abstract idea first then an example second. By having "for example" at the beginning of the sentence, we don't have to interrupt the flow of the second sentence.